### ISSUE TITLE
the question about the system

### ISSUE THREAD

**Issue by XiaoboooOvO** [WU-CVGL/SCINeRF#3]
I have a question about the data used in this model. For the original images shown in the paper, obviously, there are several frames and i suppose that there might be pixels shift between those frames. 
Have you considered to test your data on CASSI system data, which should similar to CACTI system? 
我对算法中使用的数据有疑问。对于文章中展示的原图，很显然，这是一张多帧组合的单张快照，基于其他类似的设置，我猜测帧与帧之间存在位移。请问是这样的吗？
另外，有考虑过在CASSI系统上测试多光谱单帧快照吗？这应当与CACTI系统非常类似。

**Comment by yunhaoli2020** [comment]
Correct. The temporal SCI image in this paper, i.e., the input of the system, can be regarded as the compressed multi-view images taken by a camera moving along a trajectory, as stated in this paper. 

For hyperspectral system, yes. We're now working on it.

**Comment by XiaoboooOvO** [comment]
Thank you for your reply! I have some other questions about your system.
For both hyperspectral system and the camera moving along a trajectory, I was checking some papers and the pixel shift between 2 frames is 2. From experiment side, it would be very difficult to make this value more than 1. I suppose maybe here in this paper the pixel shift is not 1? 
Of course we can do something to the DMD or the light source to make it 2 or more. Having a simulation with a larger pixel shift is fine, but for captured measurement from the SCI system, if the pixel shift is set to 1, the problem shall be harder to solve, and if the pixel shift is set to 2, here additional noise from the ignored frames will be introduced. I am not expert in this field, so I am not very sure about this. Do you have any comments on that?
